frame/democracy/src/tests.rs

```diff
@@ -0,0 +1,374 @@
+	pub const TreasuryAccount: AccountId = 0;
+}
+ord_parameter_types! {
+	pub const One: u64 = 1;
+	pub const Two: u64 = 2;
+	pub const Three: u64 = 3;
+	pub const Four: u64 = 4;
+	pub const Five: u64 = 5;
+	pub const Six: u64 = 6;
+}
+pub struct OneToFive;
+impl SortedMembers<u64> for OneToFive {
+	fn sorted_members() -> Vec<u64> {
+		vec![1, 2, 3, 4, 5]
+	}
+	#[cfg(feature = "runtime-benchmarks")]
+	fn add(_m: &u64) {}
+}
+
+impl Config for Test {
+	type Balance = u64;
+	type AssetId = u64;
+	type TreasuryAccount = TreasuryAccount;
+	type Proposal = Call;
+	type Event = Event;
+	type NativeCurrency = Balances;
+	type Currency = Tokens;
+	type EnactmentPeriod = EnactmentPeriod;
+	type LaunchPeriod = LaunchPeriod;
+	type VotingPeriod = VotingPeriod;
+	type VoteLockingPeriod = VoteLockingPeriod;
+	type FastTrackVotingPeriod = FastTrackVotingPeriod;
+	type MinimumDeposit = MinimumDeposit;
+	type ExternalOrigin = EnsureSignedBy<Two, u64>;
+	type ExternalMajorityOrigin = EnsureSignedBy<Three, u64>;
+	type ExternalDefaultOrigin = EnsureSignedBy<One, u64>;
+	type FastTrackOrigin = EnsureSignedBy<Five, u64>;
+	type CancellationOrigin = EnsureSignedBy<Four, u64>;
+	type BlacklistOrigin = EnsureRoot<u64>;
+	type CancelProposalOrigin = EnsureRoot<u64>;
+	type VetoOrigin = EnsureSignedBy<OneToFive, u64>;
+	type CooloffPeriod = CooloffPeriod;
+	type PreimageByteDeposit = PreimageByteDeposit;
+	type InstantOrigin = EnsureSignedBy<Six, u64>;
+	type InstantAllowed = InstantAllowed;
+	type Scheduler = Scheduler;
+	type MaxVotes = MaxVotes;
+	type OperationalPreimageOrigin = EnsureSignedBy<Six, u64>;
+	type PalletsOrigin = OriginCaller;
+	type WeightInfo = ();
+	type MaxProposals = MaxProposals;
+	type OriginFor = AlwaysRootOrigin;
+}
+
+pub fn new_test_ext() -> sp_io::TestExternalities {
+	let mut t = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap();
+	pallet_balances::GenesisConfig::<Test> {
+		balances: vec![(1, 10), (2, 20), (3, 30), (4, 40), (5, 50), (6, 60)],
+	}
+	.assimilate_storage(&mut t)
+	.unwrap();
+	pallet_democracy::GenesisConfig::<Test>::default()
+		.assimilate_storage(&mut t)
+		.unwrap();
+
+	orml_tokens::GenesisConfig::<Test> {
+		balances: vec![
+			(1, DEFAULT_ASSET, 10),
+			(2, DEFAULT_ASSET, 20),
+			(3, DEFAULT_ASSET, 30),
+			(4, DEFAULT_ASSET, 40),
+			(5, DEFAULT_ASSET, 50),
+			(6, DEFAULT_ASSET, 60),
+			(1, DOT_ASSET, 10),
+			(2, DOT_ASSET, 20),
+			(3, DOT_ASSET, 30),
+			(4, DOT_ASSET, 40),
+			(5, DOT_ASSET, 50),
+			(6, DOT_ASSET, 60),
+		],
+	}
+	.assimilate_storage(&mut t)
+	.unwrap();
+
+	let mut ext = sp_io::TestExternalities::new(t);
+	ext.execute_with(|| System::set_block_number(1));
+	ext
+}
+
+/// Execute the function two times, with `true` and with `false`.
+// pub fn new_test_ext_execute_with_cond(execute: impl FnOnce(bool) -> () + Clone) {
+// 	new_test_ext().execute_with(|| (execute.clone())(false));
+// 	new_test_ext().execute_with(|| execute(true));
+// }
```

Comment:
Why is this commented?

Reply:
This was commented as part of clippy error report, this function is used by one of the commented test case, will remove.

Reply:
Fixed